pyxnat/core/resources.py

```diff
@@ -156,6 +186,20 @@ def __init__(self, uri, interface):
         self._intf = interface
         self.attrs = EAttrs(self)
 
+        modules, resources = __find_all_resources__(derivatives)
+
+        for m, (mn, res) in zip(modules, resources.items()):
```

Comment:
Wouldn't be more straight forward to have `__find_all_resources__()` return just a dictionary with module (key) + list of functions (value) ?

Returning a couple of structures (a list and a dict) and zipping them afterwards is not needed IMO.

Reply:
Good catch, thanks. Fixed by 58fb13c346d80c2.